crates/db_views_actor/src/community_view.rs

```diff
@@ -286,13 +317,38 @@ mod tests {
     let local_user_form = LocalUserInsertForm::test_form(inserted_person.id);
     let local_user = LocalUser::create(pool, &local_user_form, vec![]).await?;
 
-    let new_community = CommunityInsertForm::new(
-      inserted_instance.id,
-      "test_community_3".to_string(),
-      "nada".to_owned(),
-      "pubkey".to_string(),
-    );
-    let inserted_community = Community::create(pool, &new_community).await?;
+    let inserted_community = [
```

Comment:
```suggestion
    let inserted_communities = [
```

Reply:
Good catch, though had to update the references too. Applied!